### PR TITLE
Serve static files #17

### LINKED ISSUE #17: Serve the css file for the statement page
The question statement page requires css files. Those files are not served by the server at the moment. We need to provide the css files just like we are providing the inputs for the questions. Maybe we can move both static files like css and input files to a single folder and serve that folder location?

### PR DESCRIPTION
**Pull request by HeroicHitesh**
# Thank You for your contribution to the *PyJudge*

- [ ] My title is a short description of what I am trying to do
- [ ] I have mentioned all the issues I am fixing
- [ ] Proposed changes
trying to fix #17

### CODE REVIEW

**Comment by HeroicHitesh** [comment]
@HeroicHitesh Please check it out

File: serve_css.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+@app.get('/question/<number>')
+def question(number):
+    statement = questions[number].statement
+    return template('index.html', question_number=number, question=statement)
+    return template('index.css', question_number=number, question=statement)
```
Comment (theSage21) [inline serve_css.py]: you can only return once from a function. The other returns will never be executed.

**Comment by HeroicHitesh** [comment]
I have tried to resolve the issue see if it is correct

**Comment by theSage21** [comment]
Is this running code?

**Comment by HeroicHitesh** [comment]
> 
> 
> Is this running code?

Didn't got the question. Can you elaborate?

**Comment by theSage21** [comment]
What happens when you do `python serve_css.py`?

**Pull request closed without merging**